java/images/jboss/Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM jboss/base-jdk:8
 
+ARG EXPOSE_PORT=8080
```

Comment:
Where would you set this Docker build arg with which value ? This Dockerfile is only used for building the builder image (in advance) and is completely unconnected to the apps it is supposed to build. So when building the builder image you can not reliable 'guess' all the ports an application build with this builder image will ever use.

8080 might be a good guess, but e.g. any spring-boot app just can set `server.port=8181` and you would be lost. A build arg `EXPOSE_PORT` doesnt' help here, because the image is already built at that time.